api_contracts/contracts/spendingExplorer/SpendingExplorer.md

```diff
@@ -0,0 +1,132 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Spending Explorer
+
+These endpoints are used to power USAspending.gov's Spending Explorer. This data can be used to drill down into specific subsets of data by level of detail.
+
+# Group General Spending
+
+The Spending Explorer requires the top-level (or entry point) filter to be one of three types:
+
+* Budget Function (`budget_function`)
+* Agency (`agency`)
+* Object Class (`object_class`)
+
+## General Spending Explorer [/api/v2/spending/]
```

Comment:
I'm getting a 404 / "Endpoint not found" for http://localhost:5000/api/v2/spending/